marketplace/deployer_util/provision.py

```diff
@@ -53,6 +53,16 @@ def process(schema, values, deployer_image, deployer_entrypoint):
   app_name = get_name(schema, values)
   namespace = get_namespace(schema, values)
 
+  # Inject DEPLOYER_IMAGE property values if not already present.
+  for key, value in values.items():
```

Comment:
I believe this has a bug.

We need to iterate over `schema.properties` and fine the `DEPLOYER_IMAGE` propertis. `values` wouldn't have that key. Then:

```
if key not in values:
  values[key] = deployer_image
```